internal/cloudsql/refresh.go

```diff
@@ -103,10 +105,26 @@ func fetchMetadata(ctx context.Context, client *sqladmin.Service, inst connName)
 	return m, nil
 }
 
-// fetchEphemeralCert uses the Cloud SQL Admin API's createEphemeral method to create a signed TLS
-// certificate that authorized to connect via the Cloud SQL instance's serverside proxy. The cert
-// if valid for approximately one hour.
-func fetchEphemeralCert(ctx context.Context, client *sqladmin.Service, inst connName, key *rsa.PrivateKey) (c tls.Certificate, err error) {
+func refreshToken(ts oauth2.TokenSource, tok *oauth2.Token) (*oauth2.Token, error) {
+	expiredToken := &oauth2.Token{
+		AccessToken:  tok.AccessToken,
+		TokenType:    tok.TokenType,
+		RefreshToken: tok.RefreshToken,
+		Expiry:       time.Time{}.Add(1), // Expired
```

Comment:
Why do we add one here?

Reply:
The token validation check within `ReuseTokenSource` assumed a zero value of time is not-expired. https://cs.opensource.google/go/x/oauth2/+/master:token.go;l=127